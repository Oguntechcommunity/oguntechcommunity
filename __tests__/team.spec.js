import * as React from "react";
import { mount } from "enzyme";
import Team from "../pages/team.js";

describe("Pages", () => {
	describe("Team", () => {
		it("should match snapshot", () => {
			const wrap = mount(<Team />);
			expect(wrap).toMatchSnapshot();
		});
	});
});
