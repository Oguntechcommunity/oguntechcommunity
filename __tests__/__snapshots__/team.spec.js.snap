// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Team should match snapshot 1`] = `ReactWrapper {}`;
